redux-persist-transform-compress/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for redux-persist-transform-compress 4.1
+// Project: https://github.com/rt2zz/redux-persist-transform-compress
+// Definitions by: Karol Janyst <https://github.com/LKay>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { PersistConfig, PersistTransformer } from "redux-persist";
+
+export = createCompressor;
+declare function createCompressor (config?: PersistConfig): PersistTransformer;
+declare namespace createCompressor {}
```

Comment:
Was there any discussion about that? I'd personally prefer ES6'ish way of importing modules as many people do, I think.